test/problems/hello/submissions/accepted/test-hello-lf-cr.c

```diff
@@ -1,8 +1,6 @@
 /*
  * This should give CORRECT or WRONG-ANSWER on the default problem
```

Comment:
move this file to rejected/?

Reply:
It is always `ACCEPTED` in BAPCtools because it does not have that kind of white-space config (it always normalises whitespace, I think). But moving it to `mixed/` doesn't hurt indeed :slightly_smiling_face: